osu.Game.Rulesets.Mania/Objects/Drawables/DrawableManiaHitObject.cs

```diff
@@ -23,6 +23,11 @@ public abstract partial class DrawableManiaHitObject : DrawableHitObject<ManiaHi
 
         protected readonly IBindable<ScrollingDirection> Direction = new Bindable<ScrollingDirection>();
 
+        /// <summary>
+        /// If timing based note colouring is used, the snap divisor of the note. 0 otherwise.
+        /// </summary>
+        public readonly Bindable<int> SnapDivisor = new Bindable<int>();
```

Comment:
I'm not 100% on exposing the snap divisor here rather than the colour. There's precedent for exposing colours on the `DrawableHitObject` (`AccentColour`) after all.

@mcendu any reason for going in this direction instead?

Reply:
To allow Argon to use its own colors for snap divisors, instead of using the editor snap color scheme for all skins.

Reply:
Hmm, I think at some point the colours would be defined by the skin, but the skin can still give the colours to the DHO rather than individual components.

Does argon already have custom colours somewhere? or is this for future direction?

Reply:
I think you might have missed my point. If we use accent colour, we wouldn't need the `SnapDivisor` bindable in the first place.